src/main/java/com/mturko/bestshop/BestShopApplication.java

```diff
@@ -1,17 +1,37 @@
 package com.mturko.bestshop;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.apache.catalina.Context;
+import org.apache.catalina.LifecycleException;
+import org.apache.catalina.startup.Tomcat;
+
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
 
 public class BestShopApplication {
 
-	public static void main(String[] args) {
-		ApplicationContext appContext = new ClassPathXmlApplicationContext(
-				"spring-config.xml");
-		Logger logger = LoggerFactory.getLogger(BestShopApplication.class);
-		logger.info("test");
+	public static void main(String[] args) throws LifecycleException, ServletException {
+		Tomcat tomcat = new Tomcat();
+
+		//Set Port #
+		tomcat.setBaseDir("D:\\Projects\\bestShop\\target");
+		tomcat.setPort(8080);
+		tomcat.setHostname("localhost");
+
+		String contextPath = "/";
+		String warFilePath = "D:\\Projects\\bestShop\\target\\bestShop-0.0.1-SNAPSHOT.war";
```

Comment:
It seems that we are moving to the wrong direction. The application becomes tight coupled with infrastructure. Please refer to the war file as a resource available on `classpath:` on application startup.

TBD: https://tomcat.apache.org/maven-plugin-2.0/tomcat7-maven-plugin/usage.html